seedutil/README.md

```diff
@@ -0,0 +1,8 @@
+# seedutil
+This utility is used to derive Ethereum keystore file from [aezeed](https://github.com/lightningnetwork/lnd/tree/master/aezeed) generated mnemonic seed.
```

Comment:
used to derive *an* Ethereum keystore file from *a* generated mnemonic seed.

/grammarnit